e2e/test/iothub/method/MethodFaultInjectionTests.cs

```diff
@@ -219,7 +219,7 @@ private async Task ServiceSendMethodAndVerifyResponseAsync(string deviceName, st
                     var directMethodRequest = new DirectMethodRequest()
                     {
                         MethodName = methodName,
-                        Payload = respJson,
+                        Payload = reqJson,
```

Comment:
The request should have reqJson as the payload. Looks like this was a typo while refactoring.